src/main/java/uk/ac/ebi/ena/webin/cli/WebinCli.java

```diff
@@ -1,665 +1,676 @@
-            throw WebinCliException.systemError(WebinCliMessage.CLI_MISSING_OUTPUT_DIR_ERROR.text());
-        }
-
-        String[] safeDirs = getSafeOutputDirs(dirs);
-
-        Path p;
-
-        try {
-            p = Paths.get(outputDir.getPath(), safeDirs);
-        } catch (InvalidPathException ex) {
-            throw WebinCliException.systemError(WebinCliMessage.CLI_CREATE_DIR_ERROR.format(ex.getInput()));
-        }
-
-        File dir = p.toFile();
-
-        if (!dir.exists() && !dir.mkdirs()) {
-            throw WebinCliException.systemError(WebinCliMessage.CLI_CREATE_DIR_ERROR.format(dir.getPath()));
-        }
-
-        return dir;
-    }
-
-
-    public static String
-    getSafeOutputDir(String dir) {
-        return dir
-                .replaceAll("[^a-zA-Z0-9-_\\.]", "_")
-                .replaceAll("_+", "_")
-                .replaceAll("^_+(?=[^_])", "")
-                .replaceAll("(?<=[^_])_+$", "");
-    }
-
-    public static String[]
-    getSafeOutputDirs(String... dirs) {
-        return Arrays.stream(dirs)
-                .map(dir -> getSafeOutputDir(dir))
-                .toArray(String[]::new);
-    }
-}
+/*
+ * Copyright 2018-2021 EMBL - European Bioinformatics Institute
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License. You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package uk.ac.ebi.ena.webin.cli;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.nio.file.Files;
+import java.nio.file.InvalidPathException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.apache.commons.lang3.StringUtils;
+import org.fusesource.jansi.AnsiConsole;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import ch.qos.logback.classic.LoggerContext;
+import ch.qos.logback.classic.encoder.PatternLayoutEncoder;
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.core.FileAppender;
+import ch.qos.logback.core.OutputStreamAppender;
+import de.vandermeer.asciitable.AT_Renderer;
+import de.vandermeer.asciitable.AsciiTable;
+import de.vandermeer.asciitable.CWC_FixedWidth;
+import de.vandermeer.skb.interfaces.transformers.textformat.TextAlignment;
+import picocli.CommandLine;
+
+import uk.ac.ebi.embl.api.service.SequenceToolsServices;
```

Comment:
- WebinCli.java all file line endings have changed (did not check from which encoding to which).